src/megameklab/com/util/ImageHelper.java

```diff
@@ -228,21 +228,18 @@ public static File getFluffFile(Entity unit, String dir) {
         }
 
         path = new File(path, dir).getAbsolutePath();
-        f = new File(path, unit.getShortNameRaw() + ".png");
-        if (f.exists()) {
-            return f;
-        }
-        f = new File(path, unit.getShortNameRaw() + ".jpg");
-        if (f.exists()) {
-            return f;
-        }
-        f = new File(path, unit.getShortNameRaw() + ".jpeg");
-        if (f.exists()) {
-            return f;
+        final String [] EXTENSIONS = { ".png", ".PNG", ".jpg", ".JPG", ".jpeg", ".JPEG", ".gif", ".GIF" };

```

Comment:
It's good enough for now. Ideally, there'd be some mechanism to map a specific unit type or chassis to a picture, but that's way out of scope for this.